CONTRIBUTING.md

```diff
@@ -224,69 +223,6 @@ Follow the guidelines for setting up a development environment, verifying the ch
 and submitting a pull request, as described in the [main section about updating an
 exercise test suite](#updating-an-exercise-test-suite).
 
-## Tweaking a README
```

Comment:
https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/fixing-exercise-readmes.md contains less and different information about READMEs.